### DISCUSSION TITLE
Set the more torrent providers

### DISCUSSION THREAD

**Discussion by manikandan-kyyba** [KiraLT/torrent-stream-server#222] [Ideas]
Use this NPM [https://www.npmjs.com/package/torrent-search-api](https://www.npmjs.com/package/torrent-search-api) for torrent providers.
or else use core level or some other library.

**Comment by KiraLT** [comment]
The issue with this library is that it extracts too little information about the torrent. But in any case, adding new providers is not that hard, do you know some good torrent providers which we should consider? Keep quality over the quantity.

**Comment by omgbox** [comment]
> The issue with this library is that it extracts too little information about the torrent. But in any case, adding new providers is not that hard, do you know some good torrent providers which we should consider? Keep quality over the quantity.

zooqle.com , highly recommended yts.mx
torrent.extto.com, also very good
 
rarbg.to
torlock.com
limetorrents.info

maybe add a global search function , send search query to all the providers

**Comment by KiraLT** [comment]
> maybe add a global search function, send a search query to all the providers

I am planning it, but the issue is with categories. Without categories, it's hard to find something. So we need to have global categories mapping for each provider. In any case, this feature should be available soon.
